docs/core/INPUT.md

```diff
@@ -0,0 +1,60 @@
+# Input
+
+## Input System
+In Bebone you can use event-based input system to add some actions to your keys. Input class — is a singleton class, so you can get it's instance anywhere in your codebase.
+
+Input system works with ```Action<>``` delegate. Because of this feature, you will subscribe/unsubscribe your keyboard keys and mouse buttons to the 'in-game actions' (such as jumping, running, etc.). You can do this with these two methods:
```

Comment:
Maybe it would be better to change this text to:

Input system works with ```Action<>``` delegate. With this feature you can bind/unbind you keyboard keys and mouse buttons to functor that will be executed when the bound button is pressed or released. This can be achieved using two methods: